Validation/SiTrackerPhase2V/plugins/Phase2TrackerValidateDigi.cc

```diff
@@ -107,13 +116,12 @@ void Phase2TrackerValidateDigi::analyze(const edm::Event& iEvent, const edm::Eve
   iEvent.getByToken(simVertexToken_, simVertices);
 
   // Tracker Topology
-  iSetup.get<TrackerTopologyRcd>().get(tTopoHandle_);
 
   edm::ESWatcher<TrackerDigiGeometryRecord> theTkDigiGeomWatcher;
 
   edm::ESHandle<TrackerGeometry> geomHandle;
   if (theTkDigiGeomWatcher.check(iSetup)) {
-    iSetup.get<TrackerDigiGeometryRecord>().get(geomType_, geomHandle);
+    geomHandle = iSetup.getHandle(geomToken_);
```

Comment:
as this is retrieved per event, while the esConsume is declared to be at beginRun:
```
geomToken_(esConsumes<TrackerGeometry, TrackerDigiGeometryRecord, edm::Transition::BeginRun>()),
```
this leads to the error in the relvals
```
---- Begin Fatal Exception 18-Nov-2020 11:13:55 CET-----------------------
An exception of category 'ESGetTokenWrongTransition' occurred while
   [0] Processing  Event run: 1 lumi: 1 event: 1 stream: 0
   [1] Running path 'validation_step13'
   [2] Calling method for module Phase2TrackerValidateDigi/'otDigiValid'
Exception Message:
The transition ID stored in the ESGetToken does not match the
transition where the token is being used. The associated record
type is: TrackerDigiGeometryRecord
For producers, filters and analyzers this transition ID is
set as a template parameter to the call to the esConsumes
function that creates the token. Event is the default transition.
Other possibilities are BeginRun, EndRun, BeginLuminosityBlock,
or EndLuminosityBlock. You may need multiple tokens if you want to
get the same data in multiple transitions. The transition ID has a
different meaning in ESProducers. For ESProducers, the transition
ID identifies the function that produces the EventSetup data (often
there is one function named produce but there can be multiple
functions with different names). For ESProducers, the ESGetToken
must be used in the function associated with the ESConsumesCollector
returned by the setWhatProduced function.
----- End Fatal Exception -------------------------------------------------
```